src/renderer/renderer.ts

```diff
@@ -8,5 +8,7 @@ export function renderCode<View>(templatePath: string, view: View): string {
   return Mustache.render(template, view, (partialName) => {
     const partialPath = path.join(directory, `${partialName}.mustache`);
     return fs.readFileSync(partialPath).toString();
+  }, {
+    escape: (value: string) => value,
```

Comment:
Could we only override `=`?

Reply:
The default escape is used for html, which is [escapeHtml](https://github.com/janl/mustache.js/blob/972fd2b27a036888acfcb60d6119317744fac7ee/mustache.js#L71), so it should be removed safely